<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>502</width>
    <height>222</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="autoFillBackground">
   <bool>false</bool>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="solverButton">
    <property name="geometry">
     <rect>
      <x>296</x>
      <y>10</y>
      <width>91</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Start Solver</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>14</y>
      <width>141</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="text">
     <string>audiodg.exe process</string>
    </property>
   </widget>
   <widget class="QListWidget" name="logList">
    <property name="geometry">
     <rect>
      <x>16</x>
      <y>50</y>
      <width>471</width>
      <height>131</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="trayButton">
    <property name="geometry">
     <rect>
      <x>396</x>
      <y>10</y>
      <width>91</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Tray Mod</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>502</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="githubMenu">
    <property name="title">
     <string>Menu</string>
    </property>
    <addaction name="separator"/>
    <addaction name="actionGithub"/>
   </widget>
   <addaction name="githubMenu"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionGithub">
   <property name="text">
    <string>GitHub</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
